apps/mobile/src/components/CommunitiesFollowList/CommunityFollowList.tsx

```diff
@@ -0,0 +1,51 @@
+import { FlashList, ListRenderItem } from '@shopify/flash-list';
+import { useCallback } from 'react';
+import { useFragment } from 'react-relay';
+import { graphql } from 'relay-runtime';
+
+import { CommunityFollowListFragment$key } from '~/generated/CommunityFollowListFragment.graphql';
+
+import { CommunityFollowCard } from './CommunityFollowCard';
+
+type ContractAddress = {
+  address: string | null;
+  chain: string | null;
+};
+
+type CommunityFollowListProps = {
+  communityRefs: CommunityFollowListFragment$key;
+  onLoadMore?: () => void;
+  onCommunityPress: (contractAddress: ContractAddress) => void;
+};
+
+export function CommunityFollowList({
```

Comment:
this should probably be renamed to something like `CommunityList` because it doesn't involve follows.

the `UserFollowList` which it looks like this file was based on shows a list of users next to  follow buttons
![IMG_1874](https://github.com/gallery-so/gallery/assets/80802871/caa8b4cc-fd85-4bb9-8bda-c87d5ebce3aa)

Reply:
sure ser! in the pursuit of keeping the names familiar i forgot the essence of the components.. 😅

Reply:
let's make sure the username/descriptions (and also community name / community descriptions) are vertically centered against the PFP!